Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+Callout.swift

```diff
@@ -255,12 +255,23 @@ extension BrowserViewController {
   
   private func presentVPNUpdateBillingCallout(skipSafeGuards: Bool = false) {
     if !skipSafeGuards {
-      // TODO: Condition
-      return
+      // Checking subscription receipt is in retry period
+      guard let receiptStatus = Preferences.VPN.vpnReceiptStatus.value else {
+        return
+      }
+      
+      if receiptStatus != BraveVPN.ReceiptResponse.Status.retryPeriod.rawValue {
+        return
+      }
     }
     
     presentVPNChurnPromoCallout(for: .updateBillingExpired) {
-      // TODO: Action
+      // Opens Apple's 'manage subscription' screen
+      guard let url = URL(string: "https://apps.apple.com/account/subscriptions") else { return }
```

Comment:
Maybe it's time to export this url to a shared constant? It is used in the vpn settings screen as well

Reply:
Makes sense, adde new category, fixed at https://github.com/brave/brave-ios/pull/8040/commits/c03a4a93ea069a6bb9a076b7442e09809f318308